frodo.js

```diff
@@ -2,43 +2,42 @@
  * Extras Plugin
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?